alerter.py

```diff
@@ -1,25 +1,31 @@
 alert_failure_count = 0
-
+Threshold_temp = 100
+def network_alert_main_code(celcius):
+    print(f'ALERT: Temperature is {celcius} celcius')
+    # Return 200 for ok
+    # Return 500 for not-ok
+    # stub always succeeds and returns 200 :
+    if celcius < 100 :
+        return 200
+    else:
+        return 500
 def network_alert_stub(celcius):
     print(f'ALERT: Temperature is {celcius} celcius')
+    # This is the real production code for network_alert
     # Return 200 for ok
     # Return 500 for not-ok
-    # stub always succeeds and returns 200
-    return 200
+    if celcius < 100:
+        return 200
+    else :
+        return 500
 
-def alert_in_celcius(farenheit):
+def alert_in_celcius(farenheit,network_alerter_code):

```

Comment:
Good idea to take the variable component (stub or production) as a function pointer :+1: 
Next step is to separate the test code and production code into different files.